swift/.gitignore

```diff
@@ -1,2 +1,3 @@
 /docs
 ice-test.xcodeproj
+ice.xcodeproj
```

Comment:
Do we really have an ice-test.xcodeproj?

Reply:
The rake script generates ice-test.xcodeproj when ICE_BIN_DIST is set. We can probably simplify this now and always generate ice.xcodeproj.